autonomy/chain/service.py

```diff
@@ -44,11 +44,16 @@
 from autonomy.chain.mint import transact
 
 
-DEFAULT_DEPLOY_PAYLOAD = "0x"
+DEFAULT_DEPLOY_PAYLOAD = "0x0000000000000000000000000000000000000000f48f2b2d2a534e402487b3ee7c18c33aec0fe5e4000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
This will work for Gnosis Chain only. Each chain has a different address for the default compatibility handler. Please sync with Aleks to ensure the right address is used based on the chain in quesiton

Reply:
Ok for now its the same everywhere that we support: https://github.com/safe-global/safe-deployments/blob/main/src/assets/v1.3.0/compatibility_fallback_handler.json